x-pack/plugins/security_solution/common/api/detection_engine/model/schemas.ts

```diff
@@ -107,3 +110,13 @@ export const alert_tags = t.type({
 });
 
 export type AlertTags = t.TypeOf<typeof alert_tags>;
+
+export const alert_assignees = t.type({
+  assignees_to_add: t.array(t.string),
```

Comment:
Should we use non empty string type? So you can't send `['']`.

Reply:
done